bin/views/newform.html

```diff
@@ -5,7 +5,7 @@
         <script src="/assets/scripts/newform.js" defer></script>
     </head>
     <body>
-        <form action="/new" method=POST id=post-snippet>
+        <form action="/new" method=POST id=post-snippet enctype="multipart/form-data" accept-charset=utf-8>
```

Comment:
I don't want we force multipart, this is just hiding the bug under the carpet.

Reply:
Do you have another solution? Without the multipart the error persists.

Reply:
Fix the controller code

Reply:
`.decode('ascii').encode('utf8')` ?